internal/service/remotecfg/remotecfg.go

```diff
@@ -131,9 +136,22 @@ func New(opts Options) (*Service, error) {
 		return nil, err
 	}
 
+	systemAttrs, err := getSystemAttributes()
+	if err != nil {
+		return nil, err
+	}
+
 	return &Service{
-		opts:   opts,
-		ticker: jitter.NewTicker(math.MaxInt64-baseJitter, baseJitter), // first argument is set as-is to avoid overflowing
+		opts:        opts,
+		systemAttrs: systemAttrs,
+		ticker:      jitter.NewTicker(math.MaxInt64-baseJitter, baseJitter), // first argument is set as-is to avoid overflowing
+	}, nil
+}
+
+func getSystemAttributes() (map[string]string, error) {
```

Comment:
Just a minor note: I noticed that the function signature includes an error return value, but it seems that it always returns nil. Is there a specific reason for this, or could the error be omitted?

Reply:
that's a relic from when I had wired in host. I'll clean it